portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,54 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  margin-top: 30;
+  font-family: 'Courier New', Courier, monospace;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+.slides {
```

Comment:
nit: add newlines between declarations, and spaces before curly braces in this file